prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1308,6 +1336,9 @@ int rxr_ep_init(struct rxr_ep *ep)
 err_free_tx_entry_pool:
 	if (ep->tx_entry_pool)
 		ofi_bufpool_destroy(ep->tx_entry_pool);
+err_free_rx_readcopy_pool:
```

Comment:
This whole chain of labels is overkill.  Why have multiple jump points, if you are also going to conditional test before free?

Reply:
I agree, but it probably is better to make this change in a separate commit.